src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -51,16 +51,16 @@
     private Long typeId;
     private Long priorityId;
     private Integer actionIdOnSuccess;
-
+    private Boolean newIssue;
```

Comment:
This works, but I think a slightly more descriptive field will help other logic blocks make more sense.  Maybe just `forceNewIssue`?